drivers/w1/w1_zephyr_gpio.c

```diff
@@ -48,11 +48,11 @@ LOG_MODULE_REGISTER(w1_gpio, CONFIG_W1_LOG_LEVEL);
 /*
  * Standard timing between communication operations:
  */
-#define W1_GPIO_TIMING_STD_A 6u
+#define W1_GPIO_TIMING_STD_A 1u
 #define W1_GPIO_TIMING_STD_B 64u
 #define W1_GPIO_TIMING_STD_C 60u
 #define W1_GPIO_TIMING_STD_D 10u
-#define W1_GPIO_TIMING_STD_E 9u
+#define W1_GPIO_TIMING_STD_E 5u
```

Comment:
Reducing these timings may break other targets, please see the comment below.
But, when we avoid the re-configurations it might not be needed anyways.